demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -625,6 +655,89 @@ static void handleIncomingPublish( MQTTPublishInfo_t * pPublishInfo,
 
 /*-----------------------------------------------------------*/
 
+static void updateSubscribeStatus( MQTTPacketInfo_t * pPacketInfo )
+{
+    uint8_t * pPayload = NULL;
+    size_t pSize = 0;
+
+    MQTTStatus_t mqttStatus = MQTT_GetSubAckStatusCodes( pPacketInfo, &pPayload, &pSize );
+
+    /* MQTT_GetSubAckStatusCodes always returns success if called with packet info
```

Comment:
This comment is not necessary as the API of MQTT_GetSubackStatusCodes provides information about when the function returns success. 
This comment lacks the information that success is not returned when invalid SUBACK packet is provided to the API; thus, suggest that no comment about when success is returned be mentioned here

Reply:
I disagree. It's impossible for MQTT_GetSubackStatusCodes to not return success in these demos, and it's helpful to indicate why that assumption can be made.